tests/tests.py

```diff
@@ -537,7 +537,57 @@ def test_res_comp_with_no_podpolicies(self):
                 removed = True
             else:
                 time.sleep(1)
+    
+    def test_appstatus_plugin(self):
+        kubeplus_home = os.getenv("KUBEPLUS_HOME")
+        provider = kubeplus_home + '/kubeplus-saas-provider.json'
+        
+        def cleanup():
+            cmd = "kubectl delete -f ../examples/multitenancy/hello-world/hs1.yaml --kubeconfig=%s" % provider
+            TestKubePlus.run_command(cmd)
+            cmd = "kubectl delete -f ../examples/multitenancy/hello-world/hello-world-service-composition.yaml --kubeconfig=%s" % provider
+            TestKubePlus.run_command(cmd)
+
+        if not TestKubePlus._is_kubeplus_running():
+            print("KubePlus is not running. Deploy KubePlus and then run tests")
+            sys.exit(0)
+
+        if os.getenv("KUBEPLUS_HOME") == '':
+            print("Skipping test as KUBEPLUS_HOME is not set.")
+            return
+
+        # register HelloWorldService API
+        cmd = "kubectl create -f ../examples/multitenancy/hello-world/hello-world-service-composition.yaml --kubeconfig=%s" % provider
```

Comment:
Change this to:
hello-world-service-composition-local.yaml.

hello-world-service-composition uses the 0.0.2 version of the chart.

hello-world-service-composition-local uses the 0.0.3 version of the chart.

The 0.0.3 version of the chart has replicas field in values.yaml